kubelet/datadog_checks/kubelet/common.py

```diff
@@ -104,6 +106,15 @@ def get_container_label(labels, l_name):
         return labels[l_name]
 
 
+def get_prometheus_url(default_url):
+    kubelet_conn_info = get_connection_info()
+    kubelet_conn_info = {} if kubelet_conn_info is None else kubelet_conn_info
+    error = kubelet_conn_info.get("err")
+    if error:
+        log.warning(error)
```

Comment:
@AliDatadog @iliakur @sblumenthal since we've upgraded to this version we are getting this error:
```
  File "/opt/datadog-agent/embedded/lib/python3.9/site-packages/datadog_checks/kubelet/common.py", line 114, in get_prometheus_url
    log.warning(error)
NameError: name 'log' is not defined
```

Seems like log is not defined here because it's only defined in the `except:` scope here: https://github.com/DataDog/integrations-core/blob/master/kubelet/datadog_checks/kubelet/common.py#L11-L18